schema/va-spec/aac-2017/profile-source.yaml

```
@@ -6,185 +6,616 @@ description: >-
-            evidence. The validity of this proposition, and the level of confidence/evidence supporting it,
-            may be assessed and reported by the Statement.
-        strength:
-          description: >-
-            The strength of support that the Statement is determined to provide for or against the Diagnostic Proposition
-            for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
-            Guidelines.
-          properties:
-            primaryCoding:
-              allOf:
-                - $ref: "/ga4gh/schema/gks-core/1.0.0/json/Coding"
-                - properties:
+      - $ref: "/ga4gh/schema/va-spec/1.0.1/base/json/Statement"
+      - properties:
+          proposition:
+            $ref: "/ga4gh/schema/va-spec/1.0.1/base/json/VariantClinicalSignificanceProposition"
+            description: >-
+              A proposition about a diagnostic association between a variant and condition, for which the study provides
+              evidence. The validity of this proposition, and the level of confidence/evidence supporting it,
+              may be assessed and reported by the Statement.
+          hasEvidenceLines:
+            type: array
+            ordered: false
+            items:
+              anyOf:
+                - allOf:
+                    - $ref: "/ga4gh/schema/va-spec/1.0.1/base/json/EvidenceLine"
+                    - properties:
+                        targetProposition:
+                          anyOf:
+                            - $ref: "/ga4gh/schema/va-spec/1.0.1/base/json/VariantPrognosticProposition"
+                            - $ref: "/ga4gh/schema/va-spec/1.0.1/base/json/VariantDiagnosticProposition"
+                            - $ref: "/ga4gh/schema/va-spec/1.0.1/base/json/VariantTherapeuticResponseProposition"
+                        strengthOfEvidenceProvided:
+                          properties:
+                            primaryCoding:
+                              allOf:
+                                - $ref: "/ga4gh/schema/gks-core/1.0.0/json/Coding"
+                                - properties:
+                                    code:
+                                      enum:
+                                        - A
+                                        - B
+                                        - C
+                                        - D
+                                    system:
+                                      const: AMP/ASCO/CAP Guidelines, 2017
+                - $ref: "/ga4gh/schema/gks-core/1.0.0/json/iriReference"
+          strength:
+            description: >-
+              The strength of support that the Statement is determined to provide for or against the Diagnostic Proposition
+              for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
```

Kori to drop any instances of AAC in descriptions

```suggestion
              for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP 2017
```